### ISSUE TITLE
WSL is not installed, so could not detect WSL profiles

### ISSUE THREAD

**Issue by amcoleman13** [AdamGagorik/pydarkstar#47]
I'm a little stumped, I had everything working this morning, I was able to fill my AH with goods, and had the broker up and running for a while. I was curious to try and scrub Asura server's AH, so I successfully ran the scrub app and it created the new csv file. But ever since then when I try to run any tool, I get the error message below. I have tried to remove pydarkstar and re-clone, removed the environment in conda and rebuilt from scratch, but still can't figure out how to resolve this error:

[main 2023-10-15T02:40:39.491Z] update#setState idle
[main 2023-10-15T02:40:40.489Z] WSL is not installed, so could not detect WSL profiles
[main 2023-10-15T02:40:46.180Z] Extension host with pid 16320 exited with code: 0, signal: unknown.

I can confirm I am navigating to the correct directory, my local server is up and running, the environment is active, but any tool I try to run throws this error before any kind of process..

**Comment by AdamGagorik** [comment]
It sounds like you are using windows subsystem for linux?

https://stackoverflow.com/questions/71103966/wsl-vscode-command-returning-error-not-found

<img width="734" alt="image" src="https://github.com/AdamGagorik/pydarkstar/assets/5294486/07a034b8-8c24-4128-8161-c1fd9085b29f">

**Comment by amcoleman13** [comment]
Thank you for the response Adam. I don't have WSL enabled on my environment currently which is why it was weird to be referencing that. 

I did resolve this issue and got the broker working again. I went semi-nuclear and started uninstalling/reinstalling programs to rule out any broken files and after I uninstalled VS Code it started working again immediately! I'm still unsure what running the scrubbing script changed around to cause that, but that did the trick.
